sample_data_config/twitter-ads-api/twitter-ads-api.config.yaml

```diff
@@ -69,15 +69,48 @@ twitterAdsApi:
         parameterValues:
           fromBigQuery:
             projectName: 'elife-data-pipeline'
-            # query has been filtered for now because of api rate limit 
             sqlQuery: |-
-              SELECT DISTINCT t_data.id AS entity_id, CAST(CAST(t_data.created_at AS DATE) AS STRING) AS start_date
-              FROM `elife-data-pipeline.hc_dev.v_latest_twitter_campaign_details_elife_journal`
-              JOIN UNNEST(data) AS t_data
-              WHERE -- the campaigns before '2015-01-02' has to be filtered because of api limits
-              CAST(t_data.created_at AS DATE) > CAST('2015-01-02' AS DATE)
-              AND t_data.id IN ('i61cm')
-          maxPeriodInDays: 90
+              WITH t_all_campaigns AS(
```

Comment:
For the view as discussed:

- First transform 7 (or less) days to individual days
- Then select latest record partitioned by `entity_id`, `placement` and "date", sorted by imported timestamp (DESC)

Reply:
First transform 7 (or less) days to individual days -> I couldn’t manage to create this for empty nested arrays. So continuing to use start_time for now.